salt/reactors/edx/pulsar_mitx.sls

```diff
@@ -0,0 +1,8 @@
+pulsar_mitx:
+  local.slack.post_message:
+    - tgt: {{ data['id'] }}
```

Comment:
This will cause the slack message to be posted by the minion which we don't necessarily want. We probably want the message to be sent from the master so that we don't have to add the api token to the pilllar data for all of the minions.